STM32Project/Core/Inc/projectConstants.h

```diff
@@ -73,15 +138,21 @@
 //Values needed for controlling steering 
 #define JOYSTICK_MIN -255
 #define JOYSTICK_MAX 255
-#define CCR_FOR_STEERING_MIN 1
-#define CCR_FOR_STEERING_MAX 90
+#define CCR_FOR_STEERING_MIN 72
+#define CCR_FOR_STEERING_MAX 144
```

Comment:
Rename `CCR_FOR_STEERING_...` to `LAND_CCR_FOR_STEERING_...` similar to `LAND_BRAKING_`